src/Controls/src/Core/Shapes/RoundRectangleGeometry.cs

```diff
@@ -109,5 +111,24 @@ Geometry GetRoundRectangleGeometry()
 
 			return roundedRectGeometry;
 		}
+
+		public override void AppendPath(Graphics.PathF path)
+		{
+			double density = 1.0d;
+#if ANDROID
+			density = DeviceDisplay.MainDisplayInfo.Density;
```

Comment:
Why do we only want this for Android ?

Reply:
Hey Rui, we talked about this, could you take a look again?. Thanks!